### DISCUSSION TITLE
İBXC rutbeleri

### DISCUSSION THREAD

**Discussion by hsynkader** [IBAX-io/go-ibax#704] [Soru Cevap - Türkçe]
Selamlar, rutbe arttırma işlemi nasıl olacak. Neye göre rutbem artacak. Github bağlantısı yaptım ve piyade rutbesi geldi. Bunu nasıl yükseltebilirim

**Comment by Skips1** [comment]
Puanına göre rütbe alıyorsun sanırım

**Reply by hsynkader** [reply]
> Puanına göre rütbe alıyorsun sanırım

Github daki etkinliğine göre ama neler yapmak lazım acaba. Sadece yorum ve soru sormak yeter li mi?

**Reply by Yusufknk** [reply]
Puanlarınız ne kadar çok artarsa o düzeyde rütbe de atlarsınız, buda daha çok kazanç demektir.

**Reply by hsynkader** [reply]
Teşekkürler
